packages/teams-js/src/public/people.ts

```diff
@@ -7,6 +7,9 @@ import { errorNotSupportedOnPlatform, FrameContexts } from './constants';
 import { ErrorCode, SdkError } from './interfaces';
 import { runtime } from './runtime';
 
+/**
+ * Allows your application to add a people picker to your app to let users search for and select people in their organization.
```

Comment:
>application [](http://example.com/codeflow?start=15&length=11)

You say "application" here but "app" later. We should be consistent about the terminology.

You could probably avoid using the word twice by rewording like this:
```
Allows your app to add a people picker enabling users to search for and select people in their organization.
``

Reply:
I won't lie Github Co-pilot auto generated that oops. Thank you I'll update this